index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile version skeleton</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="navbar">
+  <nav class="container">
+    <a  class="logo" href="#">My logo</a>
+    <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+  </nav>
+</header>
+<section class="hero">
+  <div class="container">
+  <h2 class="primary-text">I’m Asiati Magana <br> Glade to see you!</h2>
+
+  </div>
+
+  <p class="secondary-text"> 
+    I’m a software developer! I can help you build a product ,
+    feature or website Look through some of my work and experience!
+     If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p> 
+  
+  <h4 class="headline">LET’S CONNECT</h4>
+  <ul class="headline-img">
+    <li><a href="#"><img class="t-social-media" src="img/twitter.png" alt="twitter"></a></li>
+    <li><a href="#"><img class="l-social-media" src="img/linkedin_icon.png" alt="linkedIn"></a></li>
+    <li><a href="#"><img class="m-social-media" src="img/medium.png" alt="medium"></a></li>
+    <li><a href="#"><img class="g-social-media" src="img/github.png" alt="github"></a></li>
+    <li><a href="#"><img class="v-social-media" src="img/vec.png" alt="vector"></a></li>
+  </ul>
+</section>
+
+<section>
+  <div class="cards-container">
+    <ul class="cards-list flex-center-culomn">
+      <li class="card-item flex-center-culomn">
```

Comment:
- [ ] Please, Kindly change the background color of your project cards so that they could look like the FIGMA design. 
### Figma design:
![image](https://user-images.githubusercontent.com/71874425/159132793-9ae9d078-db81-4149-b09f-6c20e0a85aa2.png)
### Your design:
![image](https://user-images.githubusercontent.com/71874425/159132940-d8d583e2-a7b3-40b5-b5eb-9b28fe677a29.png)

Kindly fix it to:  `background-color : #ffffff;`